### ISSUE TITLE
Add a IntToTimestamp Expression

### ISSUE THREAD

**Issue by SimonMellerin** [makinacorpus/php-query-builder#5]
Could be good to have something to convert an int to a timestamp. That could be help to generate random date from an int for example, and this will be needed for the DbToolsBundle :innocent:

**Comment by pounard** [comment]
Good Idea, but I am unsure about vendors timestamp support ! Ne aware that it's also impossible to have date under 1970 with timestamp.

**Comment by SimonMellerin** [comment]
Huum, underlying subject is to generate random Date or Datetime, have you an idea on how we can do this ?

(not an easy one for sure :/ )

**Comment by pounard** [comment]
I think that issue #8 would more relevant to us, I will do it first.
